FileIO/TetGenInterface.cpp

```diff
@@ -132,7 +132,7 @@ bool TetGenInterface::parseSmeshFacets(std::ifstream &input,
 	const unsigned offset = (_zero_based_idx) ? 0 : 1;
 	std::vector<std::size_t> idx_map;
 
-	for (std::size_t k(0); k<nFacets && !input.fail(); k++)
+	for (int k(0); k<nFacets && !input.fail(); k++)
```

Comment:
Unfortunately a new warning occurs but the run time errors has gone. I will reconsider the simple fix again.